test.sh

```diff
@@ -25,11 +25,17 @@ else
 	PYTHON="docker run --rm -it -v $(pwd):/minikube -w /minikube python python"
 fi
 
-# Run "go test" on packages that have test files.
+
+# Run "go test" on packages that have test files.  Also create coverage profile
 echo "Running go tests..."
 cd ${GOPATH}/src/${REPO_PATH}
-TESTS=$(go list -f '{{ if .TestGoFiles }} {{.ImportPath}} {{end}}' ./...)
-go test -v ${TESTS}
+rm -f out/coverage.txt
+echo "mode: count" > out/coverage.txt
+for pkg in $(go list -f '{{ if .TestGoFiles }} {{.ImportPath}} {{end}}' ./...); do
```

Comment:
Could you add some comments explaining how this works? The tail -n +2 specifically.

It also might be better to move the coverage stuff to a separate script, and give it a "make coverage" command, then invoke that from Travis as well. WDYT?